CONTRIBUTING.md

```diff
@@ -0,0 +1,148 @@
+# Contributing to Eclipse Tractus-X
+
+Thanks for your interest in this project.
+
+## Project description
+
+The companies involved want to increase the automotive industry's
+competitiveness, improve efficiency through industry-specific cooperation and
+accelerate company processes through standardization and access to information
+and data. A special focus is also on SMEs, whose active participation is of
+central importance for the network's success. That is why Catena-X has been
+conceived from the outset as an open network with solutions ready for SMEs,
+where these companies will be able to participate quickly and with little IT
+infrastructure investment. Tractus-X is meant to be the PoC project of the
+Catena-X alliance focusing on parts traceability.
+
+* https://projects.eclipse.org/projects/automotive.tractusx
+
+## Project licenses
+
+The Tractus-X project uses the following licenses:
+
+* Apache-2.0 for code
+* CC-BY-4.0 for non-code
+
+## Terms of Use
+
+This repository is subject to the Terms of Use of the Eclipse Foundation
+
+* https://www.eclipse.org/legal/termsofuse.php
+
+## Developer resources
+
+Information regarding source code management, builds, coding standards, and
+more.
+
+* https://projects.eclipse.org/projects/automotive.tractusx/developer
+
+Getting started:
+
+* https://eclipse-tractusx.github.io/docs/developer
+
+The project maintains the source code repositories in the following GitHub organization:
```

Comment:
this isn't up to date, see https://github.com/eclipse-tractusx/portal/blob/main/CONTRIBUTING.md#developer-resources

Reply:
addressed with: https://github.com/eclipse-tractusx/ssi-dim-wallet-stub/pull/1/commits/743c7888797ad2c4b7da827b08b69436f02c4169